v1-2/learn/how-to-generate-code-for-protocol-buffers.md

```diff
@@ -53,18 +53,21 @@ message HelloResponse {
 }
 ```
 
-* You can run the following command to generate the client/service stub and service template.
+* You can run the following command to generate the service template file.
 ```
 $ ballerina grpc --input helloworld_service.proto --mode service --output service
 ```
-Once you execute the command, the stub file(`helloworld_service_pb.bal`) and the service template file (`helloWorld_sample_service.bal`) are generated inside the service directory.
+Once you execute the command, the service template file (`helloWorld_sample_service.bal`) are generated inside the service directory.
+Note: If you have multiple services in Protocol Buffers definition, this command will generate a stub file with common message types and service template file for each service definition. This is to avoid duplicating message types in all service files.
```

Comment:
```suggestion
> **Note:** If you have multiple services in a Protocol Buffers definition, this command will generate a stub file with common message types and a service template file for each service definition. This is to avoid duplicating message types in all the service files.
```